### ISSUE TITLE
p-menu Keyboard Navigation broken

### ISSUE THREAD

**Issue by rli-topdev** [primefaces/primeng#19313]
### Describe the bug

When using a `p-menu` in PrimeNG 20, you could navigate to the next menu item with the arrow keys. In PrimeNG 21, this no longer works. The problem is reproducible on the official documentation.

I tracked down the issue to this commit ff18adaece37f166d68eb16fb3f3efc083820e30 when removing the `[pBind]` again the navigation is working again.

### Reason for not contributing a PR

- [ ] Lack of time
- [x] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/asdmhygm-yepctx6k

### Environment

Windows 11
Firefox and Chrome

### Angular version

21.0.0

### PrimeNG version

v21

### Node version

24.13.0

### Steps to reproduce the behavior

1. Create a p-menu
2. Click on The first item to make sure the menu has focus
3. Press Arrow Down to get to the next item
-> Not working

### Expected behavior

After pressing the arrow key, the next item should be focused, as it was in PrimeNG20.
